### ISSUE TITLE
Using "padding = same " in a dau_conv2d_tf layer ??

### ISSUE THREAD

**Issue by cloudscapes** [skokec/DAU-ConvNet#3]
Hi, 
Can I use "padding = same" in a dau_conv2d_tf layer the way one can usually use it in a regular Keras/tensorflow layer(Conv2D(filters,(num_row, num_col), strides=strides, padding="same")) ?

cheers, 
H

**Comment by skokec** [comment]
Hi, 

For use in Keras you need to use the class version `DAUConv2dTF(filters, dau_units, max_kernel_size)`, which only works as `padding = same` and cannot be changed. There is actually no `padding` parameter at all.

Best,
Domen

**Comment by cloudscapes** [comment]
Thank you very much for your quick response Domen :)
So if I understand you correctly, I don't even need to specify to the DAUCOnv2dTF layer the padding = 'same' argument? Cause it already does that implicitly(default behavior)?  
cheers, 
H

**Comment by skokec** [comment]
That is correct.

Best,
Domen
